packages/stacked_services/example/lib/app/routes.dart

```diff
@@ -7,12 +7,22 @@ import '../ui/views/second_screen.dart';
 
 // Defining routes and global transitions
 @CustomAutoRouter(
-    transitionsBuilder: TransitionsBuilders.zoomIn, durationInMilliseconds: 400)
-class $Router {
-  @initial
-  HomeScreen homeScreenRoute;
-
-  FirstScreen firstScreenRoute;
-
-  SecondScreen secondScreenRoute;
-}
+    routes: <AutoRoute>[
```

Comment:
Hey, @sebastianbuechler great work, can you adjust the indentation here in routes?

Reply:
Sure, done.